src/main/kotlin/org/rust/ide/intentions/UnwrapLambdaExprIntention.kt

```diff
@@ -0,0 +1,32 @@
+package org.rust.ide.intentions
+
+import com.intellij.codeInsight.intention.PsiElementBaseIntentionAction
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import org.rust.lang.core.psi.RustBlockExprElement
+import org.rust.lang.core.psi.RustElementFactory
+import org.rust.lang.core.psi.RustLambdaExprElement
+import org.rust.lang.core.psi.util.parentOfType
+
+class UnwrapLambdaExprIntention : PsiElementBaseIntentionAction() {
+    override fun getText() = "Unwrap lambda expression body in braces"
```

Comment:
What do you think about changing this to _Remove braces from lambda expression_? Just like in [Kotlin plugin](https://github.com/JetBrains/kotlin/blob/master/idea/src/org/jetbrains/kotlin/idea/intentions/RemoveBracesIntention.kt)